src/main/java/org/openhab/tools/analysis/checkstyle/EshInfXmlValidationCheck.java

```diff
@@ -164,28 +162,31 @@ private void checkBuildProperties() {
     }
 
     @Override
-    protected void checkConfigFile(File xmlFile) throws CheckstyleException {
+    protected void checkConfigFile(FileText xmlFileText) throws CheckstyleException {
+        File xmlFile = xmlFileText.getFile();
         addToEshFiles(xmlFile);
         validateXmlAgainstSchema(xmlFile, configSchemaFile);
     }
 
     @Override
-    protected void checkBindingFile(File xmlFile) throws CheckstyleException {
+    protected void checkBindingFile(FileText xmlFileText) throws CheckstyleException {
+        File xmlFile = xmlFileText.getFile();
```

Comment:
Is it expected that we go back to the File in this class?

Reply:
I think it is. The other implementation of this method needs FileText after the changes.